JS/Closures/example2.html

```diff
@@ -21,16 +21,18 @@
  
   function functionB() { // Define a function and create a closure
     var B = 'B'
-    console.log(A, B, S);
+    console.log(A, B, G);
   }
   
-  functionB();  // prints A, B, S
+  functionB();  // prints A, B, G
     // functionB closure does not get discarded
   A = 42;
-  functionB();  // prints 42, B, S
+  functionB();  // prints 42, B, G
 }
 functionA();
+
+
 </script>
```

Comment:
Always create a new file.
Move the above code to a new file and link it on this page.

Reply:
sure i will